ReSharper.FSharp/src/FSharp.Psi.Features/src/ExtendSelection/FSharpExtendSelectionProvider.fs

```diff
@@ -28,7 +28,18 @@ type FSharpExtendSelectionProvider(settingsStore: ISettingsStore) =
 
             null
 
+        | :? IChameleonExpression as chameleonExpression ->
+            let typeInherit = TypeInheritNavigator.GetByCtorArg(chameleonExpression)
```

Comment:
Hm. I think we should not work with the chameleon node directly here.
Add a new getter to `typeInherit` grammar rule to get the inner expression. Look at something like `topBinding`.
Then you'll be able to move it to `:? ISynExpr` part.